node/typings/brand.d.ts

```diff
@@ -0,0 +1,27 @@
+interface Brand {
```

Comment:
There is some prettier issues in this file.

Reply:
done